rfd/0039-sni-alpn-teleport-proxy-routing.md

```diff
@@ -103,7 +103,7 @@ In order to expose WEB, Kubernetes proxy services in one teleport proxy port, th
 
 #### Setting SNI value for kubectl CLI:
 
-`tsh kube login` command generates a local kubeconfig.yaml file used during accessing teleport proxy by kubectl CLI. Additional to the current configuration the  tls-server-name  failed will be added with appropriate SNI value:
+`tsh kube login` command generates a local kubeconfig.yaml file used during accessing teleport proxy by kubectl CLI. Additional to the current configuration the  tls-server-name  field will be added with appropriate SNI value:
```

Comment:
Looks like there's some extra whitespace surrounding `tls-server-name` - mind fixing that while you're here?

Reply:
thanks for pointing it out. fixed that too, please check now.